airlock_processor/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
```

Comment:
how does it know which application insights to work with?

Reply:
isn't that usually via environment variables (i.e. via app settings)

Reply:
added